lib/application.js

```diff
@@ -56,7 +56,7 @@ app.init = function init() {
 
   this.cache = {};
   this.engines = {};
```

Comment:
This makes me think: should other things, like `this.cache` and `app.locals`, use `Object.create(null)` too?

Reply:
Yea, esp for 5, it should all be at least a null object these days.

Reply:
Would you like me to make a pull request for that?

Reply:
Sure, that would be most welcome!

Reply:
Sounds good. I'll try to do that in the next month.

Reply:
Done in #4861.